modules/osdk-monitoring-prometheus-servicemonitor-creating.adoc

```diff
@@ -11,22 +11,35 @@ newly created Service.
 
 .Prerequisites
 
-- Go-based Operator generated using the Operator SDK
-- Kubernetes-based cluster with the Prometheus Operator deployed
+* Go-based Operator generated using the Operator SDK
+* Kubernetes-based cluster with the Prometheus Operator deployed
 
 .Procedure
 
-. Add the `metrics.CreateServiceMonitor()` helper function to your Operator code,
-creating only one ServiceMonitor per application and namespace:
+* Add the `metrics.CreateServiceMonitor()` helper function to your Operator code:
 +
 [source,go]
 ----
-    import "github.com/operator-framework/operator-sdk/pkg/metrics"
+import(
+    "k8s.io/api/core/v1"
+    "github.com/operator-framework/operator-sdk/pkg/metrics"
+)
+func main() {
 
-    serviceMonitors, err := metrics.CreateServiceMonitors(config *rest.Config, ns string, services []*v1.Service) <1>
+    ...
+    // Populate below with the Service(s) for which you want to create ServiceMonitors.
+    services := []*v1.Service{}
+    // Create one `ServiceMonitor` per application per namespace.
+    // Change below value to name of the Namespace you want the `ServiceMonitor` to be created in.
```

Comment:
I think if you are using "`"ServiceMonitor"` " here you should do the same for Namespace, or just skip the "`". Both Service, ServiceMonitor and Namespace refer to resources so depending how you do this already in the docs. :)